swagger/swagger-generator/generator-core/src/main/java/org/apache/servicecomb/swagger/extend/ModelResolverExt.java

```diff
@@ -129,13 +135,32 @@ public Model resolve(JavaType type, ModelConverterContext context, Iterator<Mode
 
     checkType(type);
 
+    if (model instanceof ModelImpl) {
+      checkRepeat(type, (ModelImpl) model);
+    }
+
     // 只有声明model的地方才需要标注类型
     if (ModelImpl.class.isInstance(model) && !StringUtils.isEmpty(((ModelImpl) model).getName())) {
       setType(type, model.getVendorExtensions());
     }
     return model;
   }
 
+  private void checkRepeat(JavaType type, ModelImpl model) {
+    if (classNameMap.containsKey(model.getName())) {
+      if (!classNameMap.get(model.getName()).equals(type)) {
+        LOGGER.warn("The class " + type + " and the class " + classNameMap.get(model.getName())
```

Comment:
this will cause problem, so should throw exception?

Reply:
there is a previos demo 'demo-springmvc' always fail if i throw exception, so don't throw exception is for Compatibility.